Vagrantfile

```diff
@@ -71,7 +73,9 @@ end
 required_vagrant_version = travis_global_environment_variables['VAGRANT_VERSION']
 vagrant_version_constraint = ">= #{required_vagrant_version}"
 Vagrant.require_version vagrant_version_constraint
-@ui.info "The current Vagrant version satisfies the constraints: #{vagrant_version_constraint}"
+if ENV['VAGRANT_LOG']=='debug' or ENV['VAGRANT_LOG']=='info'
```

Comment:
I think we don't need this check. We can just use @ui.debug and print this only during debug

Reply:
@ferrarimarco that was my original idea, but @ui does not have a .debug method:

Message: NoMethodError: undefined method `debug' for #<Vagrant::UI::Colored:0x00000000051e3c00>

see also https://github.com/hashicorp/vagrant/blob/master/lib/vagrant/ui.rb

Reply:
Right! I assumed it was like a logger. Let me think about something

Reply:
I think we might try with a logger: https://github.com/hashicorp/vagrant/blob/master/plugins/commands/cloud/client/client.rb#L26

Reply:
but it's working well according to our needs... and I'm not getting what is the problem with that if condition 

if you want to investigate the logger approach, I'm happy... otherwise you can remove this feature, no problem on my side :-)

Reply:
I think it's fine as it is, but I wanted to show something without having to check if that environment variable is set

Reply:
I guess it's fine then. If it doesn't fit our future needs, we might go and change. Resolving for now.